packages/x-charts/src/ChartsTooltip/useItemTooltip.tsx

```diff
@@ -47,16 +43,32 @@ export function useInternalItemTooltip<T extends ChartSeriesType>():
     | ChartSeriesDefaultized<T>
     | undefined;
 
+  const xAxisId: AxisId | undefined =
+    itemSeries && isCartesianSeries(itemSeries) ? (itemSeries.xAxisId ?? xAxisIds[0]) : undefined;
+  const yAxisId: AxisId | undefined =
+    itemSeries && isCartesianSeries(itemSeries) ? (itemSeries.yAxisId ?? yAxisIds[0]) : undefined;
+  const zAxisId: AxisId | undefined =
+    itemSeries && 'zAxisId' in itemSeries ? (itemSeries.zAxisId ?? zAxisIds[0]) : zAxisIds[0];
+
+  const rotationAxisId: AxisId | undefined =
+    itemSeries && 'rotationAxisId' in itemSeries
+      ? ((itemSeries.rotationAxisId as never) ?? rotationAxisIds[0])
+      : rotationAxisIds[0];
+  const radiusAxisId: AxisId | undefined =
+    itemSeries && 'radiusAxisId' in itemSeries
+      ? ((itemSeries.radiusAxisId as never) ?? radiusAxisIds[0])
```

Comment:
Doesn't this work?

```suggestion
      ? ((itemSeries.rotationAxisId as AxisId | undefined) ?? rotationAxisIds[0])
      : rotationAxisIds[0];
  const radiusAxisId: AxisId | undefined =
    itemSeries && 'radiusAxisId' in itemSeries
      ? ((itemSeries.radiusAxisId as AxisId | undefined) ?? radiusAxisIds[0])
```

Reply:
I simplified this section to correspond to the current implementation